ocaml/testsuite/tests/typing-layouts-float64/parsing.ml

```diff
@@ -258,6 +258,17 @@ type t = #float;;
 Line 1, characters 10-15:
 1 | type t = #float;;
               ^^^^^
+Error: Did you mean float#?
```

Comment:
Here's my dream message:

```
Line 1, characters 9-15:
1 | type t = #float;;
             ^^^^^^
Error: float is neither a polymorphic variant nor a class type.
       Did you mean the unboxed type float#?
```

Note that I adjusted the location to include the `#`, because we want to replace `#float` with `float#`, not just `float` with `float#`.

I think this ideal should be easy to cook up.

Reply:
Yep, I'm able to generate this exact message!